README_GITFLEXIMOD.rst

```diff
@@ -80,7 +75,7 @@ checked out ctsm1.0.0 but really wanted to have ctsm1.1.0;
 you would simply do the following::
 
   git checkout ctsm1.1.0
-  ./manage_externals/checkout_externals
+  ./bin/git-fleximod
```

Comment:
this paragraph should be deleted

Reply:
I have a clarification question about this one:
Am I deleting this paragraph because the user does not need to run ./bin/git-fleximod update in this case?